### PR TITLE
Fixing missing information on the new Homepage

### PR DESCRIPTION
**Pull request by bramsuurdje**
> [!NOTE]
> We are meticulous when it comes to merging code into the main branch, so please understand that we may reject pull requests that do not meet the project's standards. It's never personal. Also, game-related scripts have a lower chance of being merged.

## Description

Fix some documentation issues as stated in #185

Fixes # (issue)

## Type of change
Please check the relevant option(s):

- [ ] Bug fix (non-breaking change that resolves an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (a fix or feature that would cause existing functionality to change unexpectedly)
- [ ] New script (a fully functional and thoroughly tested script or set of scripts.)

## Prerequisites
The following efforts must be made for the PR to be considered. Please check when completed:
- [ ] Self-review performed (I have reviewed my code, ensuring it follows established patterns and conventions)
- [ ] Testing performed (I have tested my changes, ensuring everything works as expected)
- [x] Documentation updated (I have updated any relevant documentation)

## Additional Information (optional)
Provide any additional context or screenshots about the feature or fix here.

## Related Pull Requests / Discussions

If there are other pull requests or discussions related to this change, please link them here:
- Related PR #185

### CODE REVIEW

File: json/mariadb.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
     "updateable": false,
     "privileged": false,
     "interface_port": "3306",
-    "documentation": null,
+    "documentation": "https://github.com/tteck/Proxmox/discussions/2915",
```
Comment (MickLesk) [inline json/mariadb.json:12]: https://github.com/community-scripts/ProxmoxVE/discussions/192

File: json/keycloak.json [hunk 2]
```diff
@@ -9,7 +9,7 @@
     "updateable": false,
     "privileged": false,
     "interface_port": "8080",
-    "documentation": null,
+    "documentation": "https://github.com/tteck/Proxmox/discussions/2922",
```
Comment (MickLesk) [inline json/keycloak.json:12]: https://github.com/community-scripts/ProxmoxVE/discussions/193

Comment (MickLesk) [reply json/keycloak.json:12]: Next Time, Please Migrate all Docs into our project :-)

Comment (havardthom) [inline json/keycloak.json:12]: The documentation button is now mixing official application docs and extra community-script docs (github discussions). This is fine for now, but I think eventually we should add support for having links in `notes` and move all github discussions to be direct links in `notes` instead

Comment (havardthom) [inline json/keycloak.json:12]:
```suggestion
    "documentation": "https://github.com/community-scripts/ProxmoxVE/discussions/193",
```

File: json/mariadb.json (hunk 1 above)
Comment (havardthom) [inline json/mariadb.json:12]:
```suggestion
    "documentation": "https://github.com/community-scripts/ProxmoxVE/discussions/192",
```

File: json/keycloak.json (hunk 2 above)
Comment (BramSuurdje) [reply json/keycloak.json]: 100% agree, this is something that i'm looking to add after i finished the json generation page

**Review by havardthom — APPROVED** [review]

**Review by MickLesk — APPROVED** [review]

**Review by havardthom — CHANGES_REQUESTED** [review]
reapproving to try and fix merge

**Review by havardthom — APPROVED** [review]

**Comment by havardthom** [comment]
Don't know what happened to this branch, I'm getting 
```
Merge attempt failed - Merge already in progress
```

**Comment by havardthom** [comment]
Maybe squash the commits locally and force push

**Comment by MickLesk** [comment]
> Maybe squash the commits locally and force push

Me too. Cannot merge.

**Comment by MickLesk** [comment]
Now it did work after all.

**Pull request merged**